doc/source/user_guide/window.rst

```diff
@@ -156,6 +156,16 @@ By default the labels are set to the right edge of the window, but a
    s.rolling(window=5).mean()
    s.rolling(window=5, center=True).mean()
 
+This can also be applied to datetime-like indices.
+
+.. ipython:: python
+
+    df = pd.DataFrame(
+        {"A": [0, 1, 2, 3, 4]}, index=pd.date_range("2020", periods=5, freq="1D")
+    )
+
+    df.rolling("2D", center=True).mean()
```

Comment:
can you make 2 ipython blocks, first should show df, second the df.rolling (i mean not a big deal)

Reply:
I split this one and added the `.rolling(..., center=False)` line for clarity, as in the example above.